scripts/bootstrap/source_data/integrated_culturing/README.txt

```diff
@@ -0,0 +1,47 @@
+README
+########
```

Comment:
The files in `source_data/` are just taken from the original paper's supplementary data, as-is. I've only kept the excel spreadsheets that are relevant to the study. This is pretty much only here as a reference, it's not used otherwise by the code.